contrib/devtools/packer/packer_build.sh

```diff
@@ -0,0 +1,8 @@
+#! /usr/bin/env sh
+
+docker run -v `pwd`:`pwd` -w `pwd` \
+    registry.gitlab.com/zingo-labs/zcash/packer:1.0 $1 \
```

Comment:
What is behind this URL?

Reply:
The docker image produced by the `publish_new_image.sh` script.